### PR TITLE
man: remove references to SHA1

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by tdsmith** [comment]
We generate our man page from a markdown document, so this change should be made in /usr/local/Library/Homebrew/manpages/brew.1.md instead, and then you should run `brew man` to regenerate the man page, and then commit the changes to both files.

File: share/man/man1/brew.1 [hunk 1]
```diff
@@ -146,7 +146,7 @@ The options \fB\-\-name=<name>\fR and \fB\-\-version=<version>\fR each take an a
 \fBedit\fR \fIformula\fR: Open \fIformula\fR in the editor\.
 .
 .IP "\(bu" 4
-\fBfetch [\-\-force] [\-v] [\-\-devel|\-\-HEAD] [\-\-deps] [\-\-build\-from\-source|\-\-force\-bottle]\fR \fIformulae\fR: Download the source packages for the given \fIformulae\fR\. For tarballs, also print SHA1 and SHA\-256 checksums\.
+\fBfetch [\-\-force] [\-v] [\-\-devel|\-\-HEAD] [\-\-deps] [\-\-build\-from\-source|\-\-force\-bottle]\fR \fIformulae\fR: Download the source packages for the given \fIformulae\fR\. For tarballs, also print SHA\-256 checksum\.
```
Comment (tdsmith) [inline share/man/man1/brew.1]: This was correct as written (fetch still prints SHA-1 and SHA-256).

File: share/man/man1/brew.1 [hunk 2]
```diff
@@ -98,7 +98,7 @@ If \fB\-\-quiet\fR is passed, list only the names of commands without the header
 If \fB\-\-autotools\fR is passed, create a basic template for an Autotools\-style build\. If \fB\-\-cmake\fR is passed, create a basic template for a CMake\-style build\.
 .
 .IP
-If \fB\-\-no\-fetch\fR is passed, Homebrew will not download \fIURL\fR to the cache and will thus not add the SHA\-1 to the formula for you\.
+If \fB\-\-no\-fetch\fR is passed, Homebrew will not download \fIURL\fR to the cache and will thus not add the SHA\-256 to the formula for you\.
```
Comment (tdsmith) [inline share/man/man1/brew.1]: saying "checksum" instead of "SHA-256" might be clearer but either is fine imho

**Comment by bf4** [comment]
@luzpaz you can just make your changes and force push to your fork (same branch you made this PR from) and it will just update this PR. No need to close it if you're going to make the changes @tdsmith suggested

**Pull request closed without merging**